main.go

```diff
@@ -41,10 +41,15 @@ func init() {
 		os.Exit(1)
 	}
 
-	os.Setenv("KUBECONFIG", path.Join(
+	err := os.Setenv("KUBECONFIG", path.Join(
 		config.KubeConfig[configprovider.FilePath],
 		fmt.Sprintf("%s.%s", config.KubeConfig[configprovider.FileName], config.KubeConfig[configprovider.FileType])),
 	)
+
+	if err != nil {
+		// Fail silently
+		fmt.Println(err)
```

Comment:
@utkarsh-pro can we log this error using MeshKit's logger?

Reply:
@lee, yes we can. Will move this to the main function so that it can share the common meshkit logger instance.